src/Nethermind/Nethermind.Wallet/ProtectedKeyStoreWallet.cs

```diff
@@ -46,7 +47,7 @@ public ProtectedKeyStoreWallet(IKeyStore keyStore, IProtectedPrivateKeyFactory p
             _protectedPrivateKeyFactory = protectedPrivateKeyFactory ?? throw new ArgumentNullException(nameof(protectedPrivateKeyFactory));
             _timestamper = timestamper ?? Timestamper.Default;
             _logger = logManager.GetClassLogger() ?? throw new ArgumentNullException(nameof(logManager));
-            _unlockedAccounts = new MemoryCache(nameof(ProtectedKeyStoreWallet));
+            _unlockedAccounts = new LruCache<string, ProtectedPrivateKey>(100, nameof(ProtectedKeyStoreWallet));
```

Comment:
Why 100 ?

Reply:
its just an estimate

Reply:
Whenever we introduce a constant like this, we should add a comment explaining why this specific number has been chosen. Even if it's an estimate we should still put that information.

Reply:
we potentially might want to make it configurable